examples/scripts/generate-standalone-files.mjs

```diff
@@ -171,6 +171,7 @@ ${exampleClass.example.toString()}
             }
             return specifiedEngine;
         }
+        let app;
```

Comment:
Can you check if the tween example still works? I just remember that it also declared app in global scope in a script and two app's are crashing on each other (pitfalls of UMD)

Reply:
You are right, the tween example broken. I don't see why tween.js should create a global app variable, so I have changed that. I checked all other examples and they all work now.

Reply:
Thank you, good call :+1: